packages/wallets/src/components/TradingAccountCard/TradingAccountCard.scss

```diff
@@ -12,6 +12,7 @@
         padding-inline: 1.6rem;
         $columns: 3;
         $grid-last-row-children: '&:nth-last-child(-n+#{$columns})';
+
```

Comment:
can we just get rid of the added empty spaces to reduce the number of files being changed

Reply:
Updated